packages/app/src/cli/services/dev/extension/payload/models.ts

```diff
@@ -21,42 +20,3 @@ export interface ExtensionsEndpointPayload extends ExtensionsPayloadInterface {
     url: string
   }
 }
```

Comment:
You mean should `@shopify/ui-extensions-server-kit` depend on `@shopify/app` ? No, I don't think so, but I do think it could depend on `@shopify/cli-kit` and that would be better than the approach in this PR.  

I'll work on that in a seperate PR and if we prefer that approach we can close this PR.